Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Use the base Python 3.6 image.  It is sufficient for accomplishing
+# most of the stuff, and includes some handy build tools like gcc.
+#
+# NOTE This could be changed to something like `python:latest` if we
+# find that there is no dependency on 3.6 itself.
+FROM python:3.6
```

Comment:
Yeah, but I don't think that SD would have any problems with a different libc – SD homeworks don't get into that very much. They might! But it might be worth checking out.

I'm down with alpine.